docs/elements/examples/sign-up.mdx

```diff
@@ -0,0 +1,41 @@
+---
+description: Examples of pre-built sign-up flows with Clerk Elements.
+---
+
+# Sign-up
+
+## Basic sign-up
```

Comment:
Actually, why don't we combine the sign-in and sign-up example pages into one page?
- The file can be called `sign-in-sign-up-examples.mdx`
- the H1 can be "Clerk Elements sign-in and sign-up examples".
- In the manifest, we could name the page "Sign-in and Sign-up Examples"

Reply:
The plan is to have a lot of examples in each page so we should split it up, even when it doesn't look like much right now